### ISSUE TITLE
Error when using tool - OrganizationID undefined

### ISSUE THREAD

**Issue by jimbobjim123** [cloudflare/cf-terraforming#92]
```
../../internal/app/cf-terraforming/cmd/root.go:154:13: undefined: cloudflare.UsingOrganization
../../internal/app/cf-terraforming/cmd/worker_route.go:64:53: api.OrganizationID undefined (type *cloudflare.API has no field or method OrganizationID)
../../internal/app/cf-terraforming/cmd/worker_route.go:71:40: api.OrganizationID undefined (type *cloudflare.API has no field or method OrganizationID)
../../internal/app/cf-terraforming/cmd/worker_script.go:38:9: api.OrganizationID undefined (type *cloudflare.API has no field or method OrganizationID)
```

**Comment by patryk** [comment]
Did you manually override `cloudflare-go` version? Because we made a breaking change Re: `OrganizationID`, but didn't release yet.

**Comment by patryk** [comment]
Hey @jimbobjim123, it turned out that go module support is still a bit strange. A colleague of mine has merged a fix in #95. Please try again.
